Fix community model

zuchuandatou opened the pull request.

ekate reviewed (commented):
Please remove celery files and add them to gitignore

zuchuandatou commented:
### PR Description: Community Model Update, Celerybeat Cleanup, and Pipfile Platform Support

#### 1. **Update Ultraviolet Communities Model to v13**
- Use new branch (13.0.6-role-filter) of communities model.

#### 2. **Remove `celerybeat-schedule` File**
- Deleted the `celerybeat-schedule*` file from git.
- Added `celerybeat-schedule*` to `.gitignore` for future commits.

#### 3. **Update Pipfile for Platform Compatibility**
- Updated the `Pipfile's sqlalchemy` dependency to support both **arm64** and **x86** architectures.
